sonar-db/src/main/java/org/sonar/db/permission/template/PermissionTemplateMapper.java

```diff
@@ -79,4 +75,6 @@ int countGroups(@Param("query") OldPermissionQuery query, @Param("templateId") l
   void groupsCountByTemplateIdAndPermission(Map<String, Object> parameters, ResultHandler resultHandler);
 
   List<String> selectPotentialPermissionsByUserIdAndTemplateId(@Param("userId") @Nullable Long currentUserId, @Param("templateId") long templateId);
+
+  int hasGroupPermission(@Param("templateId") long templateId, @Param("permission") String permission, @Nullable @Param("groupId") Long groupId);
```

Comment:
this method being named `hasGroupPermission` would be less confusing.
It being called "has", I can't tell which integer value represents true or false, otherwise it should return a boolean